assets/css/course-editor.scss

```diff
@@ -92,3 +93,50 @@ $primary: #6FCFB2;
 		}
 	}
 }
+
+.sensei-pricing-nudge {
+
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	gap: 1em;
+
+	&__content {
+		display: flex;
+		flex-direction: column;
+	}
```

Comment:
Do you think this CSS is still needed now that the panel could be in the sidebar only?

Reply:
Good catch!